tree/30_generic_methods/file_copy_from_local_source.cf

```diff
@@ -28,6 +28,11 @@
 
 bundle agent file_copy_from_local_source(source, destination)
 {
+  vars:
+     "class_prefix"        string => canonify("file_copy_from_local_source_${destination}");
+     "new_class_prefix"    string => canonify(join("_", "this.callers_promisers"));
+
   methods:
-      "placeholder" usebundle => file_copy_from_local_source_recursion("${source}", "${destination}", "0");
+      "copy without recursion" usebundle => file_copy_from_local_source_recursion("${source}", "${destination}", "0");
+      "new result classes"     usebundle => _classes_copy("${class_prefix}", "${new_class_prefix}");
```

Comment:
_class_copy cpy class_prefix (old value) into new_class_prefix (new value)
So the new promiser will be overriden by the old one (no way to distringuish if it is the previous call or this call that defined the _error or the _success)